mapit_gb/management/commands/mapit_UK_import_local-authority-eng-codes.py

```diff
@@ -0,0 +1,43 @@
+# coding=UTF-8
+# This script is used to import codes from the 'local-authority-eng'
+# register at https://local-authority-eng.register.gov.uk/. The actual mapping
+# between existing GSS codes and register IDs is in a 'mapping' file at
+# https://github.com/openregister/local-authority-data/blob/master/maps/gss.tsv
+# and cached in the mapit_gb/data directory.
+
+import csv
+import os
+
+from django.core.management.base import BaseCommand
+
+from mapit.models import CodeType, Code
+
+
+class Command(BaseCommand):
+    help = 'Import local-authority-eng codes'
+
+    def handle(self, **options):
+        # Add the CodeType
+        code_type, _ = CodeType.objects.update_or_create(
+            code='local-authority-eng',
+            defaults={
+                'description': "GOV.UK local-authority-eng codes"
+            }
+        )
+
+        # Add Codes
+        data_path = os.path.abspath(
+            os.path.join('mapit_gb', 'data', 'gss-to-local-authority-eng.tsv'))
```

Comment:
This means it only works if you run manage.py from inside its own directory? Could it use `__file__` in some way so that it doesn't matter - there are some other uses in this repo.